.github/workflows/codespell-private.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       # tomli should not be required for the next two steps (and make sure it's not)
       - run: pip uninstall -yq tomli
         if: ${{ matrix.no-toml == 'no-toml' }}
-      - run: codespell --check-filenames --skip="./.git/*,*.pyc,./codespell_lib/tests/test_basic.py,./codespell_lib/data/*,./example/code.c,./build/lib/codespell_lib/tests/test_basic.py,./build/lib/codespell_lib/data/*,README.rst,*.egg-info/*,pyproject-codespell.precommit-toml,./.mypy_cache"
```

Comment:
Please remove this change. It would be OK in a dedicated merge request, but is unrelated to inline ignores.

Reply:
Ouch! My wrong! Removing `./junit-results.xml` results in a truckload of errors. Not sure why, as this seems orthogonal to your change.

Any way, I am sorry about it, and I suggest you put `./junit-results.xml` back in the skip list.

Reply:
I explain in the commit ([b8183ee](https://github.com/codespell-project/codespell/pull/2400/commits/b8183ee98fcc55e74a8cd5b3b1fa6b4cba628fa1)) why I need to skip this file.  🤔